src/bindings/python/src/openvino/frontend/pytorch/torchdynamo/backend.py

```diff
@@ -117,7 +117,7 @@ def fx_openvino(subgraph, example_inputs, options):
         executor_parameters = None
         inputs_reversed = False
         openvino_model_caching = _get_model_caching(options)
-        if openvino_model_caching is not None:
+        if openvino_model_caching is not None and openvino_model_caching:
```

Comment:
is it possible to write tests what utilize this functionality?

Reply:
Hey Ilya, As this is a backend that's registered into torch, we use this in the torch application like this:
model = torch.compile(model, backend="openvino", options={"model_caching": 1})

For testing whether the options values are properly being sent, do you suggest we add some logging and check the logs in the test ?